lib/sqlalchemy/orm/mapper.py

```diff
@@ -1863,8 +1864,10 @@ def _property_from_column(self, key, prop):
                             "force this column to be mapped as a read-only "
                             "attribute." % (key, self, c)
                         )
+                if mc.computed is not None:
+                    kwargs["raise_on_set"] = True
```

Comment:
This should be generalized, maybe by adding a method in the colum class, so it's easier in the future to specify if a column is read only, without a list of `if ... or ... or ..`